tests/periph_spi/main.c

```diff
@@ -179,8 +203,12 @@ int cmd_init(int argc, char **argv)
         case 3: spiconf.clk = SPI_CLK_5MHZ;   break;
         case 4: spiconf.clk = SPI_CLK_10MHZ;  break;
         default:
+#ifdef CLOCK_APB1
```

Comment:
Better use something generic like. `HAS_ARBITRARY_BUS_SPEED` that gets selected together with the arch specific help text function.